backend/development.ini.sample

```diff
@@ -376,6 +376,12 @@ api.key = %(basic_setup.api_key)s
 ## TLS usage to communicate with your LDAP server
 ; ldap_tls = False
 
+## Read only fields per auth_type, parameter name is user.profile.read_only_fields.{auth_type}
+# value is a comma separated list of read-only fields.
+# example:
+# user.profile.read_only_fields.saml=username,email,public_name
+# user.profile.read_only_fields.ldap=username,password,email,public_name
+
```

Comment:
Missing default value (look in begining of sample file "Tracim Config File" about syntax).
Is it important to show default value set in tracim code.

Reply:
Maybe add a comment about why set this parameter is "important".

Reply:
Currently, default value is empty. How should I specify this ?

Reply:
Added a comment explaining why and when this parameter is important. Still don't know how to address blank default values

Reply:
resolved in last commit